src/modal/docs/readme.md

```diff
@@ -18,6 +18,7 @@ The `$modal` service has only one method: `open(options)` where available option
 * `windowTemplateUrl` - a path to a template overriding modal's window template
 * `size` - optional suffix of modal window class. The value used is appended to the `modal-` class, i.e. a value of `sm` gives `modal-sm`
 * `openedClass` - class added to the `body` element when the modal is opened. Defaults to `modal-open`
+* `appendTo` - a css selector that is supported by document.querySelector to specify the element to which the modal should be appended. Defaults to `body`
```

Comment:
Testability, and poor API signature. Angular does not work off of selectors in general, especially for arbitrary element targeting - accepting the DOM element directly in the form of a jqLite wrapped element is more in line with how one works with Angular, as it can then be consumed by a custom directive with minimal code. This would also be a cause of issues being opened due to unexpected behavior.

Testing while dealing with `document.querySelector` also would be a major pita. It is much easier to work with the desired DOM node directly, which also does not create an extra coupling of the service to `$document`.